aroquev/src/main/webapp/index.html

```diff
@@ -185,6 +185,14 @@ <h2>Comments!</h2>
       </form>
   
       <h3>Previous comments</h3>
+
+      <label for="maxComments">Change number of comments displayed (between 1 and 10):</label>
+      <input type="number" id="maxComments" name="quantity" min="1" max="10" value="5" onchange="getServletComments()">
```

Comment:
Do you think I should add this inside a form so that I can check in HTML the min and max values? Or is it okay if I do it on the JS side? I read that HTML is faster.

Reply:
What do you mean by add inside a form?

Reply:
Mmm so I apparently if I place the <input> tag inside a <form> tag, HTML can ensure that the number inserted is within the range, so I would not have to use JS to check this.
Like this
![image](https://user-images.githubusercontent.com/43384970/86620944-e95c1080-bf82-11ea-86b1-e20dfd3f2030.png)
But I find it odd that I would be using a <form> without sending a POST request anywhere...

Reply:
I think @brettapeters mentioned all the filtration, limits, etc. should be done in the server side, so the js and html just display the data. I did some work with cursors I still have some questions but maybe I can help you :)
Here's the link in case you want to check them out: https://cloud.google.com/appengine/docs/standard/java/datastore/query-cursors

Reply:
Ohhh okay, thanks @pamelalozano16 !!!

Reply:
You can trigger html input validation manually and even set custom validation messages if you wish. I made a small demo here https://codepen.io/bap13/pen/NWxYqYZ.

You can also use the CSS psuedo-class `:invalid` to give your input extra styling when it's invalid.

Reply:
Cool! I will implement it.